### DISCUSSION TITLE
Slack integration is not approved by slack?

### DISCUSSION THREAD

**Discussion by itsmnthn** [vercel/community#1186] [Help] [answered]
### Summary

I was trying to use Slack Integration to get deploy messages on slack. It is developed by Vercel Team as I can see on the slack Vercel page but it would be good if it is approved by Slack.

![Screenshot 2022-12-23 at 11 26 34 AM](https://user-images.githubusercontent.com/25129674/209279947-a400e590-600d-4932-9569-85853b860607.png)

**Comment by swarnava** [comment, marked as answer]
This is known. Our Slack integration is currently [in Beta](https://vercel.com/docs/concepts/deployments/comments/integrations#use-the-vercel-slack-beta-app)

**Reply by itsmnthn** [reply]
Oh, this makes sense. Thanks

**Reply by swarnava** [reply]
Just to add more context, this is related to our [Preview comment](https://vercel.com/docs/concepts/deployments/comments) feature.

The actual deployment integration is [generally available](https://vercel.com/integrations/slack)
